includes/alerts.inc.php

```diff
@@ -317,6 +317,11 @@ function GetContacts($results)
         }
     }
 
+    # Copy all email alerts to default contact if configured.
+    if (!isset($tmp_contacts[$config['alert']['default_mail']]) && ($config['alert']['default_copy'])) {
+	        $tmp_contacts[$config['alert']['default_mail']] = '';
```

Comment:
This has a tab in from what I can see, needs replacing for 4 spaces :)

Reply:
Oops, but I can't commit anymore: "error: failed to push some refs"

Reply:
You might need to git pull first.